### DISCUSSION TITLE
extension not working

### DISCUSSION THREAD

**Discussion by rieg-ec** [neoclide/coc.nvim#3692] [Q&A] [answered]
Hey there, i'm really new to coc and vim and was trying to become a maintainer for coc-tailwindcss, since it stopped working with new versions of coc due to some apis moving from `workspace` to `window`. 

I uploaded the changes to my [forked version](https://github.com/rieg-ec/coc-tailwindcss) and then installed the extension with vim plug. The thing is, when i run `CocCommand  tailwindCSS.headwind.sortTailwindClasses`, this is the output of `CocInfo`:
![image](https://user-images.githubusercontent.com/58893812/157668807-4deeb652-f50d-410c-817d-1a7fbf3d9300.png)
and it doesn't really does anything. There was an old issue where u guys said that not having a `lib/` directory where te plugin installed means the build failed, but i did `yarn build` and it apparently worked, so i don't really know what else to do. It'd be cool if u gave me a hand here :)

output of `yarn build`:
![image](https://user-images.githubusercontent.com/58893812/157669062-df3857cd-f751-4133-9401-671a0df0a660.png)

**Comment by chemzqm** [comment, marked as answer]
You need make sure the extension is loaded by coc.nvim https://github.com/neoclide/coc.nvim/wiki/Using-coc-extensions

**Reply by rieg-ec** [reply]
what else do i need to do if under `CoCList extensions` shows `coc-tailwindcss` with an asterisk (the one compiled by myself) ?

**Reply by xiyaowong** [reply]
https://github.com/neoclide/coc.nvim/blob/fed2dc99287b511d89faa18e7343a1cec497670c/doc/coc.txt#L3393

**Reply by rieg-ec** [reply]
nevermind, apparently updating node from 12.x to 14.x solved it, and now it is working bc i get autocomplete suggestions from the extension. Thanks!
